hudi-common/src/main/java/org/apache/hudi/common/util/HoodieRecordUtils.java

```diff
@@ -71,6 +71,11 @@ public static HoodieRecordMerger loadRecordMerger(String mergerClass) {
     }
   }
 
+  public static HoodieRecordMerger createRecordMerger(String basePath, EngineType engineType,
+                                                      String mergerImpls, String recordMergerStrategy) {
+    return createRecordMerger(basePath, engineType, ConfigUtils.split2List(mergerImpls), recordMergerStrategy);
```

Comment:
Should `HoodieTableConfig` parse the custom merge implementation class list, i.e., `HoodieTableConfig#getRecordMergerStrategy` returns the `List<String> mergerClassList`, instead of passing the config value all the way down?  and get rid of this new API.

Reply:
Fixed.